pkg/controller/nodes/task/handler.go

```diff
@@ -189,6 +195,12 @@ func (t *Handler) Setup(ctx context.Context, sCtx handler.SetupContext) error {
 		for _, tt := range p.RegisteredTaskTypes {
 			logger.Infof(ctx, "Plugin [%s] registered for TaskType [%s]", p.ID, tt)
 			t.plugins[tt] = cp
+			t.taskMetricsMap[tt] = &taskMetrics{
+				taskSucceeded: labeled.NewCounter(tt+"_success",
```

Comment:
I don't think we currently have restrictions on how to name task types... but I know prometheus does have restrictions on the character set... you might want to sanitize them before using them? I could be wrong

Reply:
done